libnmstate/nm/plugin.py

```diff
@@ -16,7 +16,12 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
+from distutils.version import StrictVersion
 from operator import itemgetter
+from warnings import warn
```

Comment:
I would prefer `logging.warning` where API user could control the output target of these warnings.
Unlike CLI tools, the library should not print to stderr unless asked explicitly.

Reply:
Done!